ElementX/Sources/Screens/RoomScreen/View/Style/TimelineItemBubbledStylerView.swift

```diff
@@ -126,35 +126,32 @@ struct TimelineItemBubbledStylerView<Content: View>: View {
             ZStack(alignment: .topLeading) {
                 contentWithReply
                     .layoutPriority(1)
-                VStack(alignment: .trailing, spacing: 0) {
-                    Spacer()
-                    HStack(alignment: .bottom, spacing: 0) {
-                        Spacer()
-                        timestamp
-                    }
-                }
+                localisedSendInfo
+                    .frame(maxWidth: .infinity, maxHeight: .infinity, alignment: .bottomTrailing)
             }
         } else {
             HStack(alignment: .bottom, spacing: 4) {
                 contentWithReply
-                timestamp
+                localisedSendInfo
             }
         }
     }
 
     @ViewBuilder
-    var timestamp: some View {
+    var localisedSendInfo: some View {
```

Comment:
As much as I hate to say it, we should use American spelling rather than British.
```suggestion
    var localizedSendInfo: some View {
```